stanford_profile_helper.module

```diff
@@ -219,6 +220,15 @@ function stanford_profile_helper_form_alter(&$form, FormStateInterface $form_sta
     // from the active filters.
     // @link https://www.drupal.org/project/views_bulk_operations/issues/3055770#comment-13116724
     unset($form['header']['views_bulk_operations_bulk_form']['select_all']);
+
+    // Sort the action menu options alphabetically.
+    $actions_array = $form['header']['views_bulk_operations_bulk_form']['action']['#options'];
+    uasort($actions_array, function ($a, $b) {
+      $a = is_a($a, "TranslatableMarkup") ? $a->render() : (string) $a;
+      $b = is_a($b, "TranslatableMarkup") ? $b->render() : (string) $b;
+      return strcasecmp($a, $b);
```

Comment:
Actually, i think you can cast `TranslatableMarkup` objects because they have a `__toString()` method. see https://git.drupalcode.org/project/drupal/-/blob/9.3.x/core/lib/Drupal/Component/Render/MarkupInterface.php#L40

Try this:
```suggestion
      return strcasecmp((string) $a, (string) $b);
```